lib/Installation/Partitioner/LibstorageNG/v4_3/CreatePartitionTablePage.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces methods to create new partition table with
```

Comment:
Please update the summary.

Reply:
I think this one is ok, was not copy/pasted.